### ISSUE TITLE
Godot Accessibility issues

### ISSUE THREAD

**Issue by ahmedthebest31** [godotengine/godot-proposals#9700]
### Describe the project you are working on

Godot editor isn't accessible at all with screen readers 
I tried every possible solution but I can't make it work 
I tried the editor on Windows, Linux and Android, of course with the same result
I also explored some of the possible solution from the community like 
Godot accessibility plugin
  without any luck 
mainly I’m using NVDA screen reader on windows and Orca screen reader on Linux
  as a blind person I really hope you can make Godot accessible to allow me to create my dream games.

### Describe the problem or limitation you are having in your project

Godot editor isn't accessible at all with screen readers 
I tried every possible solution but I can't make it work 
I tried the editor on Windows, Linux and Android, of course with the same result
I also explored some of the possible solution from the community like 
Godot accessibility plugin
  without any luck 
mainly I’m using NVDA screen reader on windows and Orca screen reader on Linux
  as a blind person I really hope you can make Godot accessible to allow me to create my dream games.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

add screen reader support

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Godot editor isn't accessible at all with screen readers 
I tried every possible solution but I can't make it work 
I tried the editor on Windows, Linux and Android, of course with the same result
I also explored some of the possible solution from the community like 
Godot accessibility plugin
  without any luck 
mainly I’m using NVDA screen reader on windows and Orca screen reader on Linux
  as a blind person I really hope you can make Godot accessible to allow me to create my dream games.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Godot editor isn't accessible at all with screen readers 
I tried every possible solution but I can't make it work 
I tried the editor on Windows, Linux and Android, of course with the same result
I also explored some of the possible solution from the community like 
Godot accessibility plugin
  without any luck 
mainly I’m using NVDA screen reader on windows and Orca screen reader on Linux
  as a blind person I really hope you can make Godot accessible to allow me to create my dream games.

### Is there a reason why this should be core and not an add-on in the asset library?

Godot editor isn't accessible at all with screen readers 
I tried every possible solution but I can't make it work 
I tried the editor on Windows, Linux and Android, of course with the same result
I also explored some of the possible solution from the community like 
Godot accessibility plugin
  without any luck 
mainly I’m using NVDA screen reader on windows and Orca screen reader on Linux
  as a blind person I really hope you can make Godot accessible to allow me to create my dream games.

**Comment by Zireael07** [comment]
It is a known issue and a PR is in the works: #76829

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot/issues/58074

This is already tracked as a bug, and there's a PR for it as you can see above

**Comment by ahmedthebest31** [comment]
How I can follow the current state of this issue?

**Comment by Zireael07** [comment]
The easiest way is to subscribe to the issue @AThousandShips linked above
